apps/sr/Testing/CMakeLists.txt

```diff
@@ -18,20 +18,20 @@ dcmqi_add_test(
   NAME ${WRITER_MODULE_NAME}_example
   MODULE_NAME ${MODULE_NAME}
   COMMAND $<TARGET_FILE:${WRITER_MODULE_NAME}>
-    --metaDataFileName ${CMAKE_SOURCE_DIR}/doc/examples/sr-tid1500-example.json
-    --outputFileName ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/sr-tid1500-example.dcm
-    --imageLibraryDataDir ${CMAKE_SOURCE_DIR}/data/ct-3slice
-    --compositeContextDataDir ${CMAKE_SOURCE_DIR}/data/sr-example
+    --inputMetadata ${CMAKE_SOURCE_DIR}/doc/examples/sr-tid1500-example.json
+    --outputDICOM ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/sr-tid1500-example.dcm
+    --imageLibraryDirectory ${CMAKE_SOURCE_DIR}/data/ct-3slice
```

Comment:
consistency and length of flags/variable names - eternal fight, at least in my head!